MAINTAINERS.md

```diff
@@ -6,28 +6,27 @@ such as architecture, process, existing issues.
-| Evgenii Mininbaev      | @l4l                     | evgenii@soramitsu.co.jp        | Security, Python library        |
-| Igor Egorov            | @igor-egorov             | igor@soramitsu.co.jp           | Development, Android library    |
-| Konstantin Munichev    | @luckychess              | konstantin@soramitsu.co.jp     | Security                        |
-| Nikita Alekseev        | @nickaleks               | alekseev@soramitsu.co.jp       | Development                     |
-| Nikolai Iushkevich     | @neewy                   | n.yushkevich@hotmail.com       | Development                     |
-| Ryu Okada              | @ryuo88                  | okada@soramitsu.co.jp          | Product vision                  |
-| Sergei Solonets        | @Solonets                | ssolonets@gmail.com            | Development                     |
-| Victor Drobny          | @victordrobny            | drobny@soramitsu.co.jp         | Development                     |
-| Vyacheslav Bikbaev     | @laSinteZ                | viacheslav@soramitsu.co.jp     | Documentation, NodeJS library   |
-| Yanno Ban              | @yannoban                | ban.yanno@nbc.org.kh           | Development                     |
+| Name                  | GitHub Id                                            | email                           | Area of expertise                                                             |
+| --------------------- | ---------------------------------------------------- | ------------------------------- | ----------------------------------------------------------------------------- |
+| Makoto Takemiya       | [@takemiyamakoto](https://github.com/takemiyamakoto) | takemiya@soramitsu.co.jp        | Product vision                                                                |
+| Ryu Okada             | [@ryuo88](https://github.com/ryuo88)                 | okada@soramitsu.co.jp           | Product vision                                                                |
+| Dmitri Venger         | [@dmitrivenger](https://github.com/dmitrivenger)     | venger@soramitsu.co.jp          | Project manager                                                               |
+| Bogdan Mingela        | [@Mingela](https://github.com/Mingela)               | mingela@soramitsu.co.jp         | Tribe Lead                                                                    |
+| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | petrosyan@soramitsu.co.jp       | Tech lead, development: Rust, C, C++                                          |
+| Dmitriy Creed         | [@Cre-eD](https://github.com/Cre-eD)                 | creed@soramitsu.co.jp           | DevSecOps                                                                     |
+| William Richter       | [@WRRicht3r](https://github.com/WRRicht3r)           | ryan.rrr66@gmail.com            | Documentation                                                                 |
```

Comment:
William, you, and I should be next to each other in the table, so that you can immediately see everyone working on the docs.